zulipterminal/helper.py

```diff
@@ -535,9 +535,18 @@ def match_emoji(emoji: str, text: str) -> bool:
 
 
 def match_topics(topic_names: List[str], search_text: str) -> List[str]:
-    return [
-        name for name in topic_names if name.lower().startswith(search_text.lower())
-    ]
+    res = []
```

Comment:
This is minor, but it's good for readability to use full words (or phrases) for variables. It can be figured out, but the less ambiguous, the quicker to read :)

Does this stand for Result? Results? Something else?

Since this represents the return value, `matches`? `matching_topics` even?